components/homme/src/theta-l_kokkos/theta_f2c_mod.F90

```diff
@@ -87,6 +87,10 @@ subroutine init_geopotential_c (ie, phis_ptr, gradphis_ptr) bind(c)
     type (c_ptr) , intent(in) :: phis_ptr, gradphis_ptr
   end subroutine init_geopotential_c
 
+  ! Uses phis to compute gradphis
+  subroutine compute_gradphis_c () bind(c)
```

Comment:
We should try to avoid modifying stuff in the homme folder. If we need to do so, we need to go through the process of "PR in E3SM" -> "merge in E3SM master" -> "upstream merge", which can be long.

Reply:
In this case, I agree that it's appropriate for this to be in the theta_f2c mod. But fortunately, this module is not _really_ needed; it's just for convenience, so one can declare all interfaces in one place, and use them via `use` directive. For this function, we can add the interface where we use it in scream...

Reply:
Yeah then, much better to add it in SCREAM. Didn't realize I could do that.